DAC/src/tlv_dac.c

```diff
@@ -96,9 +96,14 @@ static void _tlv_dac_initI2cPins()
  */
 static void _tlv_dac_initRegisters()
 {
+	//ToDo
+}
 
+void tlv_dac_I2S_write_reg(){
```

Comment:
I don't think I2S operates off of registers in the same way I2C does. Consider a different name for sending audio to the DAC.